### DISCUSSION TITLE
Add custom Headers to exported files (csv,xls)

### DISCUSSION THREAD

**Discussion by sonangrai** [zheeeng/export-from-json#125] [General]
This is a really amazing and helpfull project.
But i was thinking if we had an option to add a custom spanned header to the exported files like in the image.
It would be awesome.
<img width="382" alt="Screen Shot 2022-08-24 at 5 25 37 PM" src="https://user-images.githubusercontent.com/26194294/186409473-8dc82f8b-f7d0-4803-afdc-e904d2be9222.png">

**Comment by alpug** [comment]
I second this suggestion. The addition of a header section to list details such as the title of the table that data is coming from, and the addition of a footer section to enable the entry of copyright notices at the base of generated documents. Otherwise this ticks off most of the boxes
